config/locales/models.de.yml

```diff
@@ -1,4 +1,4 @@
-#  Copyright (c) 2012-2020, Jungwacht Blauring Schweiz. This file is part of
+#  Copyright (c) 2012-2021, Jungwacht Blauring Schweiz. This file is part of
```

Comment:
In einigen Dateien hast du den Header angepasst, in anderen nicht.

Reply:
ich habe es immer angepasst, wenn ich inhatliche Änderungen gemacht habe. Bei reinen RuboCop oder Whitespace-Commits habe ich es nicht angepasst. Würdest du das Copyright auch anpassen, wenn nur eine leere Zeile entfernt wurde? (kann ich machen, ist ne ernsthafte Frage)

Reply:
Wie fast immer ist die Anwort also "es kommt drauf an". Ich habe nicht gross nach einem Muster gesucht, nach welchem du den Header anpasst oder nicht. Deine Erklärung macht absolut Sinn und ich kann dahinter stehen. Einzig überrascht es mich, dass du von allen Leuten nicht die Dateien automatisch annotieren lässt.

Reply:
ich habe ein Makro, um es in einer Datei anzupassen. Und ich habe einen pre-commit hook, der mich warnt, wenn ich es nicht angepasst. Der Lizenzheader ist leider nicht so gleichmäßig, dass ich es immer verwende. Vermutlich wird das bald ein `autocmd`, dass beim Speichern... Warum rechtfertige ich mich eigentlich? :grin: